ansible/playbook/tasks/update.yml

```diff
@@ -150,9 +152,20 @@
         option: allow_loading_unsigned_plugins
         value: vertamedia-clickhouse-datasource
 
-    # TODO Create /srv/alertmanager/data and /srv/victoriametrics/data in pmm-managed
-    #      to support Docker-way (swap image) updates.
-    #      https://jira.percona.com/browse/PMM-7024
+    - name: Clickhouse logger         | Configure clickhouse logger
+      replace:
+        path: /etc/clickhouse-server/config.xml
+        regexp: '(<logger>[\s\S]*)</logger>'
```

Comment:
minor comment, by why the grouping here, as I don't see it used in the replace

Reply:
I was testing different stuff and forgot to revert. Fixed, thanks!